public/cli/lib/rendering/renderFrames.ts

```diff
@@ -0,0 +1,42 @@
+import { record } from './recordFrames';
+import fs from 'fs';
+import path from 'path';
+import { AnimationSettings, Timings, NotifyEvent, ProjectData } from '../../../models';
+import { allowedNodeEnvironmentFlags } from 'process';
+const _ = require('lodash');
```

Comment:
This should now use import instead of require. Also this is a very large dependency. Import only the submodules you need, e.g. `import debounce from 'lodash/debounce';`.

Reply:
done.